src/Aquifer.Jobs/appsettings.json

```diff
@@ -0,0 +1,40 @@
+{
+    "Analytics": {
+        "ApiManagementResourceId": "/subscriptions/dacfa945-984c-4708-9f2d-7d8358f223d5/resourceGroups/BiblioNexus-Global/providers/Microsoft.ApiManagement/service/aquifer-api-management",
+        "ApiRequestStatsTableName": "ApiRequestStatsQa",
+        "AppInsightsResourceId": "/subscriptions/dacfa945-984c-4708-9f2d-7d8358f223d5/resourceGroups/BiblioNexus-Dev/providers/microsoft.insights/components/appinsights-qa",
+        "CustomEventsTableName": "CustomEventsQa",
+        "HoursBetweenRuns": 1,
+        "PageViewsTableName": "PageViewsQa",
+        "StorageAccountUri": "https://aquiferstoragedev.table.core.windows.net"
```

Comment:
A bunch of these are QA specific values and should be removed (see below comment).